core/tests/protractor_utils/ExplorationPlayerPage.js

```diff
@@ -22,43 +22,48 @@ var general = require('./general.js');
 var interactions = require('../../../extensions/interactions/protractor.js');
 
 var ExplorationPlayerPage = function() {
-  var conversationInputAll =
-    element.all(by.css('.protractor-test-conversation-input'));
-  var conversationInput =
-    element(by.css('.protractor-test-conversation-input'));
+  var conversationInput = by.css('.protractor-test-conversation-input');
```

Comment:
This should be surrounded by element(), same as everything else. All "elements" should be treated the same way.

Reply:
Done